src/main/resources/application.properties

```diff
@@ -1,6 +1,6 @@
 spring.application.name=Spring-boot-web-greqit
 spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
-spring.datasource.url=jdbc:mysql://localhost:3306/add_SoppingCart_fix
+spring.datasource.url=jdbc:mysql://localhost:3306/add_order
```

Comment:
bad naming for database add_order
rename please
Did you change you DB for every HW ?

Reply:
Yes, I change DB for every HW

Reply:
rename db like - book_store for example